packages/calendar/src/components/calendar-wrapper.tsx

```diff
@@ -31,11 +31,13 @@ export default function CalendarWrapper({ $app }: props) {
     handleWindowResize($app)
   }
 
-  useEffect(() => {
-    onResize()
-    window.addEventListener('resize', onResize)
-    return () => window.removeEventListener('resize', onResize)
-  }, [])
+  if ($app.config.isResponsive) {
+    useEffect(() => {
+      onResize()
+      window.addEventListener('resize', onResize)
+      return () => window.removeEventListener('resize', onResize)
+    }, [])
+  }
```

Comment:
I believe the if-condition needs move inside the useEffect-callback. I don't think conditional effects are allowed in Preact

```suggestion

    useEffect(() => {
      if ($app.config.isResponsive) {
         onResize()
         window.addEventListener('resize', onResize)
      }
      return () => window.removeEventListener('resize', onResize)
    }, [])
```

Reply:
Good to know! Hadn't found it not to be allowed. The thought was to only set the hook if the condition is fulfilled instead of having a hook which repeatedly checks the condition.